### DISCUSSION TITLE
Is this plugin still alive?

### DISCUSSION THREAD

**Discussion by LeonardoGentile** [SkepticMystic/breadcrumbs#501] [General]
Probably not the best place where to ask but probably the most visible place for the dev and users.
The question is self-explanatory: @SkepticMystic is this plugin development still alive or the project has been abandoned?

I'm very thankful for your work and I use this plugin extensively every day but I've noticed issues are piling up and no one is responding. 

I've been investing hundreds of hours in my Obsidian structure using this plugin so I'd like to know if I should stop and rearrange my structure or look somewhere else.

Thanks! 🙏

**Comment by SkepticMystic** [comment]
EDIT: Please see my comment further below

**Comment by LeonardoGentile** [comment]
Hey @SkepticMystic thanks for your answer!
That's too bad as I seriously think this is the most useful plugin that gives Obsidian so much more value. 
But I understand, life gets in the way :) 

Everyone's brain and Obsidian's organization is different and I can't really function without a typed hierarchy as you kindly provided. 
I hope someone will take over your hard work and carry on this awesome project.

Maybe you can look for a maintainer by writing on the README or create a post on the obsidian forum?


Best of luck

**Comment by cvh-obsi** [comment]
is there a way to replace breadcrumbs by another plugin.
Me too, am using this plugin all the time (not less)
Maybe excalibrain?

**Comment by Anarchtism** [comment]
ExcaliBrain works, but doesn't have anything like tree view and doesn't have an easy way to move between "previous" and "next" notes like Breadcrumbs... Been trying to get used to it, though. Another plugin somewhat like ExcaliBrain is [Graph Link Types](https://github.com/natefrisch01/Graph-Link-Types).

**Comment by LeonardoGentile** [comment]
Excalibrain _might_ work but it lacks many features breadcrumbs provided, I've highlighted some https://github.com/zsviczian/excalibrain/issues/115

**Comment by BricoleurSoul** [comment]
### So sorry to hear this !
I'm about to embark on a new analysis project and this was the most powerful tool in my Obsidian toolbox.
**Thanks for everything you gave us!**

**Comment by SkepticMystic** [comment]
To everyone watching this issue, I've got some good news!
I'm far into the process of rebuilding Breadcrumbs from scratch.
Please read more info here: https://github.com/SkepticMystic/breadcrumbs/blob/master/V4.md
And let me know if you've got any questions or need help switching over :)

**Reply by LeonardoGentile** [reply]
You got us worried! I've been looking for an alternative for months! 😬
I honestly think this functionality should exist as a core feature of Obsidian, but it's not, so thanks for the great work! 👍

**Comment by Pummelfisch** [comment]
After almost a year since the last comment here and now 8 months since the last commit to v4, I would like to reopen this discussion. I heavily use breadcrumbs in version 4. Yet everything still works, even though with minor bugs. Still, I am wondering how long compatibility with obsidian persists and if this project is abandoned for good or if @SkepticMystic is just to busy at the moment, which would be okay too, of course.

**Reply by jacobmichels** [reply]
I too would like an update. @SkepticMystic if you're too busy to continue to actively maintain this project, you might consider sharing the maintenance burden with other willing contributors.

**Comment by SkepticMystic** [comment]
Hey everyone, thanks for the comments.
My focus is elsewhere atm, but I still intend to release v4 and fix the big issues.
I'm open to sharing maintenance with other contributors, but I've not had anyone reach out.

**Reply by mariomui** [reply]
is mprojects willing to maintain? I've been looking to use his rust rewrite for some time now.
I can't do it cuz i'm [redacted-by-self].
If possible. https://github.com/SkepticMystic/breadcrumbs/issues/567#issuecomment-2950028202  <-- is one line that stops v3 from crashing on startup. 

And i cant seem to test v4 because the documentation's too sparse. Maybe, I have a skill issue, but my tolerance for pain is fairly high so it could be a not enough documentation issue.

**Reply by churnish** [reply]
Hi, I was thinking of trying out this plugin but I'm hesitant due to v4 being in the works. Would it be best to revisit Breadcrumbs if v4 releases or dive in now? (I'm not one to beta test.)

**Reply by michaelpporter** [reply]
@SkepticMystic I'm willing to help maintain. While I do not know typescript yet, I have worked with many languages. I am currently in DevSecOps, to start with I can help test PRs and get some of the backlog updated. That process will help me learn more.

**Comment by michaelpporter** [comment]
@SkepticMystic I would like to extend an offer to help maintain Breadcrumbs.

**Reply by SkepticMystic** [reply]
Hey Michael, thanks for reaching out.
Let's chat on discord, please drop me a message there. My username is the same
